File: TIMS-X.Server/TIMS-X.Server/JsonModels/ProviderBlockSchedule.ts




    export class ProviderBlockSchedule { 
        public providerId: number;
        public scheduleBlockId: number;
        public scheduleTimeSlotId: number;
        public createdDate: Date;
        public rowVersion: number[];
        public scheduleBlock: ScheduleBlock;
        public scheduleTimeSlot: ScheduleTimeSlot;
		
        public pendingDelete: boolean;
        public id: number;
        public hasStateBeenSet: boolean;
        
    }

File: TIMS-X.Core/Domain/ProviderBlockSchedule.cs
using System;
using TIMS_X.Core.Domain.Base;

namespace TIMS_X.Core.Domain
{
    public class ProviderBlockSchedule : Entity
    {
        public int ProviderId { get; set; }
        public int ScheduleBlockId { get; set; }
        public int ScheduleTimeSlotId { get; set; }
        public DateTime CreatedDate { get; set; }
        public byte[] RowVersion { get; set; }
        public ScheduleBlock ScheduleBlock { get; set; }
        public ScheduleTimeSlot ScheduleTimeSlot { get; set; }
    }
}
